### DISCUSSION TITLE
ClamAV Flagging Node Module As Trojan

### DISCUSSION THREAD

**Discussion by BPowell76** [openstreetmap/iD#10569] [General]
I was scanning iD 2.30.4 with the latest signatures for ClamAV and it's coming back saying the `imurmurhash/imurmurhash.min.js` in node_modules is a Trojan (`PUA.Win.Trojan.Xored-1`).

Not sure if this is a false positive or not, but it might be worth looking into.

**Comment by 1ec5** [comment]
This file isn’t part of iD proper; it pulls in [imurmurhash-js](https://github.com/jensyt/imurmurhash-js/) v0.1.4, the latest version, via NPM when you install dependencies:

https://github.com/openstreetmap/iD/blob/3025d4f4217238ecd57dbb09734c9d9e93124569/package-lock.json#L3722

Fortunately, it’s a false positive: jensyt/imurmurhash-js#1.

**Reply by BPowell76** [reply]
Thanks for confirming. I suppose if ClamAV is saying it's a `PUA` it could be a false positive as it also flagged the package in other github forks I was working on.
